From 72f99b17a1c298c9f9bd3234478a84fc65f91a72 Mon Sep 17 00:00:00 2001
From: Jonathan Lichtner
 <195780826+JonathanLichtnerAMD@users.noreply.github.com>
Date: Tue, 17 Jun 2025 23:04:39 +0000
Subject: [PATCH 1/2] TMP: do not build deprecated kernels

Pragma message causes the build to fail.  This is a very short-term
workaround.
---
 include/ck/ck.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/ck/ck.hpp b/include/ck/ck.hpp
index 26e478794..fe456a991 100644
--- a/include/ck/ck.hpp
+++ b/include/ck/ck.hpp
@@ -270,7 +270,7 @@
 #endif // gfx90a
 
 // set flag to 1 to build deprecated instances
-#define CK_BUILD_DEPRECATED 1
+#define CK_BUILD_DEPRECATED 0
 
 namespace ck {
 
-- 
2.25.1

